eng/common/scripts/Test-Spelling.ps1

```diff
@@ -0,0 +1,91 @@
+[CmdletBinding()]
```

Comment:
It might be worth naming this script after what is actually doing. Something like "check-spelling-in-changed-files.ps1" or something along those lines.